src/MeetingsSDKAdapter.js

```diff
@@ -323,14 +323,13 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
   async joinMeeting(ID) {
     try {
       const sdkMeeting = this.fetchMeeting(ID);
-      const {localAudio, localVideo} = this.meetings[ID];
+      const {localVideo} = this.meetings[ID];
```

Comment:
Can you do the same change for `localVideo`? Since the logic is the same, it's bound to have the same issue as audio.
This also ensure we're consistent on how we treat local streams.

Reply:
I have updated the original commit message and updated the localVideo as well - There are some es lint formatting changes I was thinking of committing but haven't yet - let me know if I should commit those as well